packages/frontend/src/redux/actions/account.js

```diff
@@ -159,34 +189,36 @@ export const redirectToApp = (fallback) => async (dispatch, getState) => {
     }));
 };
 
-
 export const allowLogin = () => async (dispatch, getState) => {
-    const { account } = getState();
-    const { url } = account;
-    const { success_url, public_key, title } = url;
+    const accountId = selectAccountId(getState());
+    const contract_id = selectAccountUrlContractId(getState());
```

Comment:
It would be good for consistency to update the snake_case variables to camelCase now that the new selectors have abstracted away the property access.

Reply:
Right, fixed all similar issues in this file.